README.md

```diff
@@ -2,16 +2,33 @@
 
 [![Build Status](https://travis-ci.org/sonatype-nexus-community/nexus-repository-apt.svg?branch=master)](https://travis-ci.org/sonatype-nexus-community/nexus-repository-apt) [![DepShield Badge](https://depshield.sonatype.org/badges/sonatype-nexus-community/nexus-repository-apt/depshield.svg)](https://depshield.github.io)
 
+> **WARNING: 3.17.0 includes a feature flagged version of the APT plugin by default**. We're releasing this way so that 
+we can get early feedback on the APT plugin without impacting the wider user base. From 3.18.0 APT will be 
+enabled by default.
+
+>If you are an APT user upgrading to 3.17 setting ```nexus.apt.enabled=true``` in ```./etc/nexus.properties``` will 
+enable the plugin. **You will not be able to install the community plugin.**
+
+>There are some differences from the community version of the plugin. First of all, we have added component records for 
+the Debian files which adds support for features such as *Search*, *Cleanup Policies*, *Tagging (PRO only)*
+and *Moving between repositories (PRO only)*. We've also removed the "retain N versions" feature as this conflicts with 
+Cleanup Policies and future work we are doing in that space.
+
+> We've added support for other core features: *Restore Metadata Task*, *Routing Rules*, *API for Repository creation
```

Comment:
Minor: It may be unnecessary to mention Routing Rules since it's coming out with 3.17 too?

Reply:
We had to explicitly add support for it however I've just realized we haven't actually finished that so this is risky. I've removed it https://github.com/sonatype-nexus-community/nexus-repository-apt/pull/116/commits/ddb9cbb481142ff2bfed7d0760493a8fee04dc2d

Reply:
Frankly, I am not sure if it'll need to be added for each new format.  I now wonder about Go.